setup.py

```diff
@@ -48,6 +48,7 @@
         'elasticsearch',
         'numpy',
         'python-ipmi',
+        'PrettyTable'
```

Comment:
What is 'PrettyTable'?

Reply:
PrettyTable is a Python library for generating simple ASCII tables.